include/openssl/kdf.h

```diff
@@ -85,6 +85,27 @@ OPENSSL_EXPORT int SSKDF_hmac(uint8_t *out_key, size_t out_len,
                               const uint8_t *info, size_t info_len,
                               const uint8_t *salt, size_t salt_len);
 
+// KBKDF_ctr derives keying material using the KDF counter mode algorithm,
```

Comment:
> As a note for the reviewers. I'm still on the fence with the proposed `KBKDF_ctr` for the one-shot function name. It may be clearer to call it `KBKDF_counter` or even `KBKDF_hmac_counter` (as the PRF function could be implemented with KMAC if we so ever wished). Open to suggestions / discussions around the name choice.

I think your suggestion to add "hmac" to the name is a good one, especially since OpenSSL has implementation for cmac as well. `KBKDF_ctr_hmac` is what I think of. A similar combination is used in [`EVP_aead_aes_128_ctr_hmac_sha256`](https://github.com/aws/aws-lc/blob/b2bbd9c287d3fff4e209892d342ea3b0d562477e/include/openssl/aead.h#L135).

Also, I suggest you expand the acronym to Key-Based Key Derivation Function or Key-Based KDF somewhere as was done with SSKDF.